CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 0.11.1
+
+## New Features
+
+- Added head_commit property to PagurePullrequest (@jsakalos)
+
+## Fixes
+
+- Packit rev-dep tests were refactored. (@lbarcziova)
+- Descriptions in playbooks fixed. (@lbarcziova)
+- Test invocation was improved. (@ttomecek)
+- Error handling was improved. (@ttomecek)
```

Comment:
These are quite general and hence doesn't carry much info. I'd either merge into one or remove both.

Reply:
fixed